### DISCUSSION TITLE
Add Canadian waters to Deepzoom

### DISCUSSION THREAD

**Discussion by AMCarter333** [jaybo/deepzoom-blog#26] [Ideas]
I live in the Pacific Northwest near the San Juan Islands and the Canadian border. We do a lot of boating across the border in the Gulf Islands and further north along the Sunshine Coast and Desolation Sound. Since finding DeepZoom many years ago, I always wondered why it does not show currents and tides on the Canadian side of trhe border line. Is that something that is possible or coming in the future?

**Comment by jaybo** [comment]
Great question!

Until very recently, DeepZoom has been an unprofitable hobby.  The charts and tide harmonics needed to cover Canada are not in the public domain (unlike the USA), and require licensing.  The licensing fees seem to be $50 per current/tide station.  In response to your question, I just reached out the licensing agency to see if any other options are available to obtain this data.

**Reply by FastCubed** [reply]
Hey Jay,
Any updates on this?

**Comment by AMCarter333** [comment]
Jay,Thanks again for responding so quickly. That’s discouraging that Canada charges for every tide/current station info.Your site is unique. And it requires access to a decent internet signal… which can be elusive and fairly expensive for U.S. boaters traveling in BC waters. Have you considered initiating a survey with the people who use and rely on DeepZoom to see how many would be willing to pay for adding Canadian current info?We recently experimented while on our boat for a month in BC waters by buying a Canadian cellular SIM card with a 15 GB data plan for internet access. That worked fairly well but was expensive. However we see more and more people using Starlink systems now to produce reliable, high speed internet access virtually anywhere (including notorious dead zones like Princess Louisa Inlet). If this is something you feel would be valuable… have you considered mounting a campaign to persuade Canadian authorities to make the information available for free or at a lower cost to an enterprise like yours that provides a unique and valuable service to boaters?Lastly, have you considered linking up with the people who supply the “Canadian Current Atlas”? Their Atlas is extremely useful to see a big picture regarding currents hour by hour throughout the Salish Sea area. Although we use a number of mobile apps and annual books like ”Ports & Passes” for tide/current info, we find the Atlas to be one of the best resources for understanding what is likely to happen as far as currents.Cheers, Mac CarterSent from my iPhoneOn May 19, 2023, at 3:12 PM, Jay Borseth ***@***.***> wrote:﻿
Great question!
Until very recently, DeepZoom has been an unprofitable hobby.  The charts and tide harmonics needed to cover Canada are not in the public domain (unlike the USA), and require licensing.  The licensing fees seem to be $50 per current/tide station.  In response to your question, I just reached out the licensing agency to see if any other options are available to obtain this data.

**Comment by SethSea** [comment]
Hi Jay, I too am US boater who loves cruising British Columbian and would love to see BC tides/currents supported in you app. It is sad that Canada does not make this information available at no cost as in USA. Tide/current information is available on CA website but, from what I found, is a PDF format by station - not useful for programming. I use an iPhone app, AyeTides. It does have CA stations and the app is free. I, too, purchase the publication "Ports and Passes" but do not reference it as often anymore as the information is avaiable in AyeTides and the Navionics "Boating" app and my Raymarine chart plotter.  I also use the CA "Atlas of Tidal Currents", charts of tidal streams;  but it requires a separate publication to know which chart is applicable on a given date/time (so called "Washburn's Tables") which is published by year. Recently I discovered a website that has them available for download (https://borsboom.io/current-atlas-tables/). There is an app for this "PNW Current Atlas" by TinyOctopus (https://tinyoctopus.net/CurrentAtlas), which I also use. But after several years it still does not include the northern half of the CA current atlas.

Really, I am amazed by what you have done with DeepZoom and used it often on a recent cruise and intend to get more familiar with it.

**Comment by jaybo** [comment]
As of version 3.20, Canadian nautical charts (but not tides and currents) are now available. New Zealand, Australia, and Norway too!

**Comment by SethSea** [comment]
Thank you!
From: Jay Borseth ***@***.***>
Reply-To: jaybo/deepzoom-blog ***@***.***>
Date: Monday, January 26, 2026 at 10:14
To: jaybo/deepzoom-blog ***@***.***>
Cc: Seth Siegal ***@***.***>, Comment ***@***.***>
Subject: Re: [jaybo/deepzoom-blog] Add Canadian waters to Deepzoom (Discussion #26)


As of version 3.20, Canadian nautical charts are now available, as are New Zealand, Australia, and Norway.
